packages/openactive-integration-tests/test/features/payment/common.js

```diff
@@ -0,0 +1,162 @@
+const chakram = require('chakram');
```

Comment:
Note this whole file needs attention in light of the refactor, and likely most if not all of it should be generalised and live somewhere more generally available to all tests?